pyproject.toml

```diff
@@ -38,6 +38,9 @@ python = "^3.6.2"
 pyhcl = "^0.4.4"
 requests = "^2.27.1"
 
+# Note on linux krb5 libs are required: `sudo apt install libkrb5-dev` or `yum install krb5-devel`
+requests_gssapi = { version = "^1.2.0", optional = true } # kerberos support, will include `gssapi`
```

Comment:
requests_gssapi requires users to install [kfw](https://web.mit.edu/kerberos/kfw-4.1/kfw-4.1.html) which is a pain

kerberos and winkerberos do not have that requirement and would be preferable

Reply:
@masariello where is this requirement?  It's a hidden requirement for windows users?

Here are dependencies for `requests_gssapi`
```
    install_requires=[
        'requests>=1.1.0',
        'gssapi',
    ],
```

Reply:
Correct. it's dependency of gssapi ([here](https://github.com/pythongssapi/python-gssapi/blob/main/gssapi/_win_config.py#L14)).

I realized after posting the comment.

There is a conda requests-gssapi package which seems to provide the same functionality but uses native O/S krb libs

I have too say the python ecosystem is a real mess around such a crucial piece of functionality